File: scratch/indent_check.py
import sys

with open(r"c:\Users\levyc\fauves-platform\src\pages\EventPanelV2.tsx", "r", encoding="utf-8") as f:
    lines = f.readlines()

for idx in range(594, 610):
    print(f"{idx+1}: {repr(lines[idx])}")

File: scratch/indent_check.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\levyc\\fauves-platform\\src\\pages\\EventPanelV2.tsx', 'utf8');
const lines = content.split('\n');
for (let i = 594; i < 610; i++) {
    console.log(`${i+1}: ${JSON.stringify(lines[i])}`);
}
